docs/docker/building-net-docker-images.md

```diff
@@ -0,0 +1,189 @@
+---
+
+
+#Building Docker Images for .NET Core Applications
+
+In order to get an understanding of how to use .NET Core and Docker together, we must first get to know the different Docker images that are offered and when is the right use case for them. Here we will walk through the variatons offered, build an ASP.NET Core Web API, use the Yeoman Docker tools to create a debuggable container as well as peek at how Visual Studio Code can assist in the process. 
+
+## Docker image variatons
+
+The `microsoft/dotnet` images are offered in three different variations, each for a specific use case.
+
+- `microsoft/dotnet:<version>-sdk` : that is **microsoft/dotnet-preview2-sdk**, this image contains the .NET Core SDK which includes the .NET Core and Command Line Tools (CLI). Use this image for your development processes; dev, test debug.
+
+- `microsoft/dotnet:<version>-onbuild` : that is **microsoft/dotnet:onbuild**, contains `ONBUILD` triggers which should cover most applications. The build will `COPY` you application, run `dotnet restore` and create an `ENTRYPOINT dotnet run` instruction to run the application when the Docker image is run.
+
+- `microsoft/dotnet:<version>-core` : that is **microsoft/dotnet:1.0.0-core**, image contains only .NET Core (runtime and libraries) and it is optimized for running [portable .NET Core applications](https://docs.microsoft.com/en-us/dotnet/articles/core/app-types).
+
+- `microsoft/dotnet:<version>-core-deps` : that is **microsoft/dotnet:1.0.0-core-deps**, if you wish to run self-contained applications use this image.  Contains the operating system with all of the native dependencies needed bt .NET Core. Can also be used to build a custom copy of .NET core by compiling coreclr and corefx.
+
+Latest versions of each variant:
+
+- `microsoft/dotnet` or `microsoft/dotnet:latest` (includes SDK)
+- `microsoft/dotnet:onbuild`
+- `microsoft/dotnet:core`
+- `microsoft/dotnet:core-deps`
+
+Here is a list of the images after a `docker pull <imagename>` on a development machine to show the various sizes.
+
+```
+REPOSITORY          TAG                  IMAGE ID            SIZE
+microsoft/dotnet    onbuild              19b6a6c4b1db        540.4 MB
+microsoft/dotnet    1.0.0-preview2-sdk   a92c3d9ad0e7        540.4 MB
+microsoft/dotnet    core                 5224a9f2a2aa        253.2 MB
+microsoft/dotnet    1.0.0-core-deps      c981a2eebe0e        166.2 MB
+microsoft/dotnet    core-deps            c981a2eebe0e        166.2 MB
+microsoft/dotnet    latest               03c10abbd08a        540.4 MB
+microsoft/dotnet    1.0.0-core           b8da4a1fd280        253.2 MB
+```
+
+## Prerequisites
+
+- [.NET Core](http://dot.net)
+- Docker Engine : see [Docker Installation page](http://www.docker.com/products/docker)
+- Yeoman tooling for ASP.NET for creating the Web API application
+- Yeoman tools for Docker from Microsoft
+
+Install the Yeoman generators for ASP.NET Core and Docker using npm 
```

Comment:
You used "Yeoman tooling", "Yeoman tools" above and "Yeoman generators" in this instruction.  Are they the same? 

If so, you should make it clear that installing the Yeoman generators will install the Yeoman tooling and the Yeoman tools.